x-pack/plugins/security_solution/public/flyout/document_details/left/index.tsx

```diff
@@ -38,8 +40,8 @@ export const LeftPanel: FC<Partial<DocumentDetailsProps>> = memo(({ path }) => {
     'securitySolutionNotesEnabled'
   );
 
-  const visualizationInFlyoutEnabled = useIsExperimentalFeatureEnabled(
-    'visualizationInFlyoutEnabled'
+  const [visualizationInFlyoutEnabled] = useUiSetting$<boolean>(
```

Comment:
one thing we should check with @ferenrigue is which position we want the tab to be displayed at. Right now it's the first one, which means that every time a use opens the expanded section of the flyout via the `Expand details` button they will land on the `Visualize/Session View` tab. There is a chance that this tab won't have any data.
Wouldn't it make more sense to land on the `Insights/Entities` tab? That way we don't change the current behavior, and we're pretty much guaranteed to have data to show...
Just a thought!

Reply:
yes, kept it as first because it was how it was presented in the original mock (long time ago). definitely need to revisit the ordering... the order in left panel also does not align with the right side (investigations was up first in right panel).
This is why i want to merge it first, discuss them on the mock is fine, but I'm sure we will get more feedback once PM and design can access it in siem dev.
I'm leaning toward keep it as is and do a follow up PR to polish it up. wdyt?

Reply:
works for me!